test/migrations/v3-data-availability-proposal.ts

```diff
@@ -0,0 +1,125 @@
+import BNJS from 'bignumber.js';
+import { BigNumberish } from 'ethers';
+
+import config from '../../src/config';
+import { getDeployConfig } from '../../src/deploy-config';
+import { getDeployerSigner } from '../../src/deploy-config/get-deployer-address';
+import { DIP_22_IPFS_HASH } from '../../src/lib/constants';
+import { log } from '../../src/lib/logging';
+import { waitForTx } from '../../src/lib/util';
+import { impersonateAndFundAccount } from '../../src/migrations/helpers/impersonate-account';
+import { createV3DataAvailabilityProposal } from '../../src/migrations/v3-data-availability-proposal';
+import {
+  DydxGovernor__factory,
+  DydxToken__factory,
+  MockStarkPerpetual__factory,
+} from '../../types';
+import { advanceBlock, increaseTimeAndMine } from '../helpers/evm';
+
+export async function executeV3DataAvailabilityNoProposal({
+  starkwarePriorityAddress,
+  starkPerpetualAddress,
+}: {
+  starkwarePriorityAddress: string,
+  starkPerpetualAddress: string,
+}) {
+  //const deployConfig = getDeployConfig();
+  const starkwarePrioritySigner = await impersonateAndFundAccount(starkwarePriorityAddress);
+  const starkPerpetual = new MockStarkPerpetual__factory(starkwarePrioritySigner).attach(starkPerpetualAddress);
+
+  await starkPerpetual.mainAcceptGovernance();
+  //await waitForTx(await starkPerpetual.registerGlobalConfigurationChange(deployConfig.STARK_PERPETUAL_CONFIG_HASH));
```

Comment:
For my understanding why are these lines commented out?

Reply:
sorry I was playing around with testing after hitting some issues on npm run test